src/infra/services/config.ts

```diff
@@ -0,0 +1,39 @@
+import * as t from "io-ts";
+import * as E from "fp-ts/Either";
+import * as RE from "fp-ts/ReaderEither";
+import { pipe, flow } from "fp-ts/function";
+
+import { sequenceS } from "fp-ts/lib/Apply";
+
+import { IntegerFromString } from "@pagopa/ts-commons/lib/numbers";
+import { readFromNodeEnv } from "../../app/env";
+
+export const ServiceConfig = t.type({
```

Comment:
I'd rename everything to something like 'ServiceClient' or (better) 'IOApiClient'  since `Service` is too generic.